client.go

```diff
@@ -23,6 +23,7 @@ type Conn struct {
 	net.Conn                         // a net.Conn holding the connection
 	UDPSize        uint16            // minimum receive buffer for UDP messages
 	TsigSecret     map[string]string // secret(s) for Tsig map[<zonename>]<base64 secret>, zonename must be in canonical form (lowercase, fqdn, see RFC 4034 Section 6.2)
+	GSSAPI         GSSAPI            // an implementation of the GSSAPI interface
```

Comment:
```suggestion
	GSSAPI                     // An implementation of the GSSAPI interface.
```
I think we can remove the second `GSSAPI` here. While typing this I wonder if there is a better name for the interface. TsigGSS or something that points to tsig?

Reply:
I renamed the interface to `TsigGSS`.